src/java.base/share/classes/java/time/LocalTime.java

```diff
@@ -1631,7 +1631,16 @@ public int hashCode() {
      */
     @Override
     public String toString() {
-        StringBuilder buf = new StringBuilder(18);
+        var buf = new StringBuilder(18);
+        formatTo(buf);
+        return buf.toString();
+    }
+
+    /**
+     * Prints the toString result to the given buf, avoiding extra string allocations.
+     * Requires extra capacity of 10 to avoid StringBuilder reallocation.
```

Comment:
```suggestion
     * Requires extra capacity of 18 to avoid StringBuilder reallocation.
```
Sorry I think I left a typo, you see toString is `new StringBuilder(18)`